src/backlight/portfolio/portfolio.py

```diff
@@ -108,23 +118,20 @@ def fusion_positions(positions: List[Positions]) -> List[Positions]:
 
     unique_positions = []
     symbols = [p.symbol for p in positions]
-    for position in positions:
-        s = position.symbol
-        if symbols.count(s) > 1:
-            df = pd.DataFrame(
-                data=np.array([p.values for p in positions if p.symbol == s]).sum(
-                    axis=0
-                ),
-                index=position.index,
-                columns=position.columns,
-            )
-            pos = Positions(df)
-            pos.symbol = s
-            unique_positions.append(pos)
-            for _ in range(symbols.count(s)):
-                symbols.remove(s)
-        elif symbols.count(position.symbol) == 1:
-            unique_positions.append(position)
+    index = positions[0].index
+    columns = positions[0].columns
+
+    for symbol in sorted(set(symbols)):
+        df = pd.DataFrame(
+            data=np.array([p.values for p in positions if p.symbol == symbol]).sum(
+                axis=0
+            ),
+            index=index,
+            columns=columns,
+        )
+        position = Positions(df)
+        position.symbol = symbol
```

Comment:
For these lines, it is better to create `positions.from_dataframe`.

The implementation should be
in backlight.positions
```
def from_dataframe(df, symbol);
    pos = Positions(df)
    pos.reset_cols()
    pos.symbol = symbol
    return pos
```